src/components/week/calendarWeekView.component.ts

```diff
@@ -49,6 +49,7 @@ import { CalendarUtils } from '../../providers/calendarUtils.provider';
         (dayClicked)="dayClicked.emit($event)"
         (eventDropped)="eventTimesChanged.emit($event)">
       </mwl-calendar-week-view-header>
+      <div class="cal-events" [style.height]="maxHeight+'px'" [style.overflowY]="maxHeight ? 'auto' : ''">
       <div *ngFor="let eventRow of eventRows" #eventRowContainer class="cal-events-row">
```

Comment:
Rather than adding another option I'd prefer just to add the wrapping div and let users add the styles themselves through CSS.